README.md

```diff
@@ -416,38 +216,6 @@ which is one of the extra dependencies for JWT, uses OpenSSL 3.0.
 To enable FIPS mode for the `cryptography` library, you need to install a FIPS-compliant version of OpenSSL
 during the installation process of cryptography using the `pip` command.
 
-# Versions
-We use a modified version of [Semantic Versioning](https://semver.org/) for all changes. See [version strategy](VERSIONS.md) for details which is effective from 30 July 2022.
-
-## Supported Version
-
-Only the current MAJOR version of SDK is supported. New features, functionality, bug fixes, and security updates will only be added to the current MAJOR version.
-
-A current release is on the leading edge of our SDK development, and is intended for customers who are in active development and want the latest and greatest features.  Instead of stating a release date for a new feature, we set a fixed minor or patch release cadence of maximum 2-3 months (while we may release more often). At the same time, there is no schedule for major or breaking release. Instead, we will communicate one quarter in advance the upcoming breaking change to allow customers to plan for the upgrade. We always recommend that all users run the latest available minor release for whatever major version is in use. We highly recommend upgrading to the latest SDK major release at the earliest convenient time and before the EOL date.
-
-## Version schedule
-
-| Version | Supported Environments                                  | State     | First Release | EOL/Terminated |
-|---------|---------------------------------------------------------|-----------|---------------|----------------|
-| 4       | Python 3.8+                                             |           |               |                |
-| 3       | Python 3.6+                                             | Supported | 17 Jan 2022   | TBD            |
-| 2       |                                                         | EOL       | 01 Nov 2018   | 17 Jan 2022    |
-| 1       |                                                         | EOL       | 10 Feb 2015   | 01 Nov 2018    |
-
-# Contributing
-
-See
-[CONTRIBUTING.md](https://github.com/box/box-python-sdk/blob/main/CONTRIBUTING.md).
-
-## Developer Setup
-
-Create a virtual environment and install packages -
-
-``` console
-mkvirtualenv boxsdk
-pip install -r requirements-dev.txt
-```
-
 ## Testing
 
 Run all tests using -
```

Comment:
In the [sdk-gen](https://github.com/box/box-python-sdk/tree/sdk-gen) we had a whole section about running integration tests, shouldn't we add this also here?

Reply:
As discussed we remove this section for now